content/docs/guides/liquibase.md

```diff
@@ -0,0 +1,294 @@
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd
+        http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.5.xsd">
+        <changeSet author="alex" id="myIDNumber1234">
+            <createTable tableName="comments">
+                <column autoIncrement="true" name="comment_id" type="INTEGER">
+                    <constraints nullable="false" primaryKey="true" primaryKeyName="comments_pkey"/>
+                </column>
+                <column name="post_id" type="INTEGER">
+                    <constraints nullable="false" foreignKeyName="fk_comments_post_id" referencedTableName="posts" referencedColumnNames="post_id"/>
+                </column>
+                <column name="author_id" type="INTEGER">
+                    <constraints nullable="false" foreignKeyName="fk_comments_author_id" referencedTableName="authors" referencedColumnNames="author_id"/>
+                </column>
+                <column name="comment" type="TEXT"/>
+                <column name="commented_date" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP"/>
+            </createTable>
+        </changeSet>
+    </databaseChangeLog>
+    ```
+
+## Deploy your change
+
+Deploy your database schema change by running the [update](https://docs.liquibase.com/commands/update/update.html) command:
+
+```bash
+liquibase update
+```
+
+<details>
+<summary>Command output</summary>
+
+If the command was successful, you’ll see output similar to the following:
+
+```bash
+Starting Liquibase at 07:33:53 (version 4.24.0 #14062 built at 2023-09-28 12:18+0000)
+Liquibase Version: 4.24.0
+Liquibase Open Source 4.24.0 by Liquibase
+Running Changeset: dbchangelog.xml::myIDNumber1234::AlexL
+
+UPDATE SUMMARY
+Run:                          1
+Previously run:               0
+Filtered out:                 0
+-------------------------------
+Total change sets:            1
+
+Liquibase: Update has been successful. Rows affected: 1
+Liquibase command 'update' was executed successfully.
+```
+
+</details>
+
+<Admonition type="info">
+When you run a changeset for the first time, Liquibase automatically creates two tracking tables in your database:
+
+- [databasechangelog](https://docs.liquibase.com/concepts/tracking-tables/databasechangelog-table.html): Tracks which changesets have been run.
+- [databasechangeloglock](https://docs.liquibase.com/concepts/tracking-tables/databasechangeloglock-table.html): Ensures only one instance of Liquibase runs at a time.
+
+You can verify these tables were created by viewing the `blog` database on the **Tables** page in the Neon console. Select **Tables** from the sidebar.
+</Admonition>
+
+## Rollback a change
+
+Try rolling back your last change by running the Liquibase [rollbackCount](https://docs.liquibase.com/commands/rollback/rollback-count.html) command:
+
+```bash
+liquibase$ liquibase rollbackCount 1
```

Comment:
Do you need the liquibase$ command prompt here? Since we've added it to PATH...

Reply:
good point. removed.